### PR TITLE
cli: use fraggle rock runner by default, deprecate legacy navigation runner

### PR DESCRIPTION
**Pull request by adamraine**
[Doc](https://docs.google.com/document/d/1zHUX8kIboJGUFC3IujQULj50uf4P-dmlq6FPydh7uSA/edit?resourcekey=0-dYDmlElqbicfhq8zfI1ffg#heading=h.qdlstqig47z4)

### CODE REVIEW

File: lighthouse-cli/cli-flags.js [hunk 1]
```diff
@@ -124,10 +124,10 @@ function getFlags(manualArgv, options = {}) {
           type: 'boolean',
           describe: 'Pause after page load to wait for permission to continue the run, evaluate `continueLighthouseRun` in the console to continue.',
         },
-        'fraggle-rock': {
+        'legacy-navigation': {
           type: 'boolean',
           default: false,
-          describe: '[EXPERIMENTAL] Use the new Fraggle Rock navigation runner to gather results.',
+          describe: '[DEPRECATED] Use the legacy navigation runner to gather results.',
```
Comment (connorjclark) [inline lighthouse-cli/cli-flags.js:130]: Say more about why you might want to use this?

Comment (adamraine) [reply lighthouse-cli/cli-flags.js:130]: > I don't think we have any specific scenarios where FR runner would break

The new config format would break some clients. I'll add a note about that.

This might be a good time to update the config docs, so we know what our story will be.

**Comment by connorjclark** [comment]
Can you investigate the CI failures?

File: docs/recipes/custom-gatherer-puppeteer/test.sh [hunk 2]
```diff
@@ -3,6 +3,6 @@
 # Make sure we're in this `docs/recipes/customer-gatherer-puppeteer` directory
 cd "$(dirname "$0")"
 
-node node_modules/.bin/lighthouse --config-path=custom-config.js https://www.example.com --output=json |
+node node_modules/.bin/lighthouse --legacy-navigation --config-path=custom-config.js https://www.example.com --output=json |
```
Comment (adamraine) [inline docs/recipes/custom-gatherer-puppeteer/test.sh:6]: This recipe won't work in FR because it uses `driver.wsEndpoint()` which doesn't exist on the FR driver. We could add the function, or instead we could add a `driver.getPage` so users can easily get the correct puppeteer page to use.

That will be done in a separate PR as part of our docs updates.

**Review by connorjclark — APPROVED** [review]
